aws-fs-lambda-webserver/LambdaWebServer/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  }
+}
```

Comment:
Can we live without this file?

Reply:
They were created by the AWS template I guess. I've removed them